invokeai/backend/stable_diffusion/diffusers_pipeline.py

```diff
@@ -756,13 +755,19 @@ def img2img_from_latents_and_embeddings(
             return self.check_for_safety(output, dtype=conditioning_data.dtype)
 
     def get_img2img_timesteps(
-        self, num_inference_steps: int, strength: float, device
+        self, num_inference_steps: int, strength: float, device=None
```

Comment:
Is the `device` arg still needed?

Reply:
I left it there incase anyone wants to override. If it is explicitly supplied, it will override the base function.